docs/src/extend/shareable-configs.md

```diff
@@ -66,59 +72,72 @@ Be sure to replace `eslint-config-myconfig` with the actual name of your module.
 
 ## Using a Shareable Config
 
-Shareable configs are designed to work with the `extends` feature of `.eslintrc` files. Instead of using a file path for the value of `extends`, use your module name. For example:
+To use a shareable config, include the config name in the `extends` field of a configuration file. For the value, use your module name. For example:
 
 ```json
 {
     "extends": "eslint-config-myconfig"
 }
 ```
 
-You can also omit the `eslint-config-` and it will be automatically assumed by ESLint:
+You can also omit the `eslint-config-` and it is automatically assumed by ESLint:
 
 ```json
 {
     "extends": "myconfig"
 }
 ```
 
-### npm scoped modules
+You cannot use shareable configs with the ESLint CLI [`--config`](../use/command-line-interface#c---config) flag unless you explicitly specify the path to the module (e.g. `eslint --config node_modules/eslint-config-airbnb/index.js`).
```

Comment:
```suggestion
You cannot use shareable configs with the ESLint CLI [`--config`](../use/command-line-interface#-c---config) flag.
```

I'd leave just "you cannot" because this is not the intended use of shareable configs, and depends on internal structures of shareable config packages so we shouldn't recommend it in any way.